src/wasm/kotlin/java/arcs/jvm/JvmRuntimeClient.kt

```diff
@@ -17,26 +17,28 @@ actual object RuntimeClient {
     actual fun <T : Entity<T>> singletonClear(particle: Particle, singleton: Singleton<T>): Unit =
         throw NotImplementedError()
 
-    actual fun <T : Entity<T>> singletonSet(particle: Particle, singleton: Singleton<T>, encoded: String): Unit =
-        throw NotImplementedError()
+    actual fun <T : Entity<T>> singletonSet(particle: Particle,
+        singleton: Singleton<T>,
+        encoded: NullTermByteArray): Unit = throw NotImplementedError()
```

Comment:
Newline required between end of params and closing paren.

Reply:
Also, consider using `TODO` instead of `throw NotImplementedError` here and elsewhere in this class.

```kotlin
fun myFunSignature(arg1: ArgType): ReturnValue = 
    TODO(".... some reason why it's not ready yet, or when it can be implemented")
```

This not only throws `NotImplementedError`, but also will show up in IntelliJ/AndroidStudio's TODOs list.